src/platforms/android/usage/advanced-usage.mdx

```diff
@@ -162,7 +162,7 @@ when using a Android Gradle Plugin >= v3.6.0, and it needs to be enabled with th
 
 Apart from this, `sentry-native` also uses [tagged pointers](https://source.android.com/devices/tech/debug/tagged-pointers)
 internally, and will therefore not work out of the box on newer Android devices which have their own conflicting pointer tagging.
-To prevent this, disable the [`allowNativeHeapPointerTagging`](https://developer.android.com/guide/topics/manifest/application-element#allowNativeHeapPointerTagging) configuration option.
+To prevent this, disable the [`allowNativeHeapPointerTagging`](https://developer.android.com/guide/topics/manifest/application-element#allowNativeHeapPointerTagging) configuration option, you only need to do that if you are using `sentry-android` <= 3.0, `sentry-android` >= 3.1 already fixed it.
```

Comment:
```suggestion
To prevent this, disable the [`allowNativeHeapPointerTagging`](https://developer.android.com/guide/topics/manifest/application-element#allowNativeHeapPointerTagging) configuration option. You only need to do that if you are using `sentry-android` < 3.1, as this is fixed in `sentry-android` >= 3.1.
```